components/gallery/GalleryItemAction/GalleryItemAction.vue

```diff
@@ -16,9 +16,9 @@
     <!-- change price as an owner -->
     <GalleryItemPriceRelist
       v-if="isOwner && nft?.id && nft?.price && nft?.collection.id"
-      :collection-id="nft.collection.id"
-      :nft-id="nft.id"
-      :nft-price="nft.price"
+      :collection-id="nft?.collection.id"
```

Comment:
Do we still need this? since we already had `v-if="isOwner && nft?.id && nft?.price && nft?.collection.id"` on line 18

Reply:
You are right. It's just for testing.